po/de.po

```diff
@@ -1282,12 +1282,12 @@ msgstr "Tornado-Flak-Turm"
 #: data/base/messages/strings/names.txt:247
 #: data/mp/messages/strings/names.txt:288
 msgid "Proximity Bomb Turret"
-msgstr "Näherungsbombenabwurfschacht"
+msgstr "Selbstmordkommando-Bombenturm"
```

Comment:
:x:

The issue with this translation is: It's not in the original. Also, the word itself does not make much sense in German as a "Selbstmordkommando" is a group of people on a (presumably) deadly mission and "Selbstmordkommando-Bombenturm" would be a "bomb tower" of theirs. Meaning unclear.

Generally, "bomb turret" is translated as "Bombenabwurfschacht" (literally: bomb drop chute) to make it clear that it is not some kind of tower. Both, turret and tower, are translated as "Turm" in German, so some way to distinguish the meanings is needed.

The translation as suggested by you would be (literally) "suicide squadron bomb turret".

Reply:
I googled "proximity bomb" to find out how other games translate it:

* Wikipedia redirects it to ["Proximity fuse"](https://en.wikipedia.org/wiki/Proximity_fuze).
  The equivalent German article is titled ["Abstandszünder"](https://de.wikipedia.org/wiki/Abstandsz%C3%BCnder).

* In Guild Wars, Horizon Zero Dawn, Resident Evil and Star Wars the word
  means "landmine" and is probably translated as such.

An alternative name for this kind of weapon is "Nova Turret" and "Supernova Turret",
according to an [old forum post](http://forums.wz2100.net/viewtopic.php?f=6&t=4961#p50366):

> Black Project wrote:
> Proximity Bomb Turret: Never hearded and seen this weapon in action, but never was implemented. I believe that was a suicidal turret that explodes everything within range.
>
> Proximity Superbomb Turret: Same as the previous one, but a lot more powerful.
>
>> I've seen these used in mods, called "Nova Turret" and "Supernova Turret".

Reply:
Do you have in-game footage of this? Totally looked like a VTOL weapon to me. I remember 1.12 iirc had mines and they also worked, but were not called "proximity bomb".

How is that weapon supposed to work? Does it put landmines everywhere or will it just explode if something touches it?

Wikipedia calls "Abstandszünder" "Näherungszünder", which is where I got the idea for the current translation from. Do you think "Näherungsbombe" or "Näherungsmine" could work? The "turret" will not appear in the translation.

Reply:
> Do you have in-game footage of this? Totally looked like a VTOL weapon to me. I remember 1.12 iirc had mines and they also worked, but were not called "proximity bomb".
> 
> How is that weapon supposed to work? Does it put landmines everywhere or will it just explode if something touches it?

I only know what is written in the ["Non-Implemented Features List" thread](http://forums.wz2100.net/viewtopic.php?f=6&t=4961), according to which
* the weapon is distinct from landmines (those were also available)
* it exploded on impact (like [petards in Age of Empires II](https://ageofempires.fandom.com/wiki/Petard_(Age_of_Empires_II)))

> 
> Wikipedia calls "Abstandszünder" "Näherungszünder", which is where I got the idea for the current translation from. Do you think "Näherungsbombe" or "Näherungsmine" could work? The "turret" will not appear in the translation.

"Näherungsbombe" is okay, since it's a bomb, if a human-guided one.

Reply:
> In my opinion, this superfluous term should be removed from the original source (not used since **20** years) rather than discussed how it's translated.

Translating strings for unused technologies is useful for mods, which
 cannot introduce new translatable messages on their own.